android/sdk/src/main/java/org/jitsi/meet/sdk/ExternalAPIModule.java

```diff
@@ -114,6 +114,11 @@ public void sendEvent(String name, ReadableMap data, String scope) {
             dataMap.put("url", data.getString("url"));
             listener.onConferenceWillLeave(dataMap);
             break;
+
+        case "LOAD_CONFIG_ERROR":
+            dataMap.put("error", data.getString("error"));
```

Comment:
I believe that, like the conference-related events, we should provide `url` in the event (map/dictionary/object) because that's what the SDK consumers have as a means of associating a request (`loadURL`) with a response (`JitsiMeetViewDelegate`/`JitsiMeetViewListener`).

Reply:
Yep. We don't have it at that point yet though. Let's take this as a base and discuss the possibility to make this more generic and maybe turn it into a "fatal error" handler of sorts, which would include URL and reason.

Reply:
Of course we have it - it's where we call `_loadConfig` and wait for its promise to settle in order to call `setLocationURL`, we just have to put it in the action's payload. Anyway, I'll write it myself.